app/controllers/comment_controller.php

```diff
@@ -63,26 +66,103 @@ public function edit()
         $title = $thread->title;
         $page = Param::get('page_next', 'edit');
 
-
         switch ($page) {
-        case 'edit':
-            break;
-        case 'edit_end':
-           $comment->body = Param::get('body');
-
-            try {
-                $comment->edit($comment_id);
-                redirect("/comment/view?thread_id={$_SESSION['thread_id']}");
-            } catch (ValidationException $e) {
-                $page = 'edit';
-            }
-            break;  
-        default:
-            throw new NotFoundException("{$page} not found");
-            break;
+            case 'edit':
+                break;
+            case 'edit_end':
+               $comment->body = Param::get('body');
```

Comment:
Same as [this comment](https://github.com/erikjohnmoreno/board_exercises/pull/2#discussion_r24557679).